src/ApplyOfflineMap.cpp

```diff
@@ -282,6 +311,42 @@ try {
 		AnnounceStartBlock("Applying first offline map to data");
 	}
 */
+	
+		DataArray3D<int> dataGLLNodesIn;
+		DataArray3D<double> dataGLLJacobianIn;
+		
+		DataArray3D<int> dataGLLNodesOut;
+		DataArray3D<double> dataGLLJacobianOut;
+		Mesh OverlapMesh;
+		Mesh SourceMesh;
```

Comment:
Why do you need copies of the **overlap** and **source** mesh ?

Reply:
The overlap and source mesh are needed when the CAAS algorithm is used with local bounds preservation (the case when fCAASLocal = true).  Specifically, to implement the local bounds preservation, we need to find a set of source faces that cover each given target face.  This requires both the overlap mesh and the source mesh.